script/tool/lib/src/version_check_command.dart

```diff
@@ -273,6 +275,7 @@ ${indentation}HTTP response: ${pubVersionFinderResponse.httpResponse.body}
       }
     } else {
       final GitVersionFinder gitVersionFinder = await retrieveVersionFinder();
+      previousVersionSource = await gitVersionFinder.getBaseSha();
```

Comment:
This is a logical change since getBaseSha() isn't necessarily `master` (or `main`), right?  This sounds like the correct thing to do.  It might be worth mentioning this a bit more explicitly in the description in case it does cause some problems.

Reply:
> This is a logical change since getBaseSha() isn't necessarily `master` (or `main`), right?

In that the error logging was wrong before, and now isn't, yes. `getBaseSha()` is what the code that populates `previousVersion` actually uses; the error logging on failure (which is all this is used for) was incorrectly hard-coding that to `master`. In practice, in presubmit, it's the same thing, so it hadn't been noticed.

> It might be worth mentioning this a bit more explicitly in the description in case it does cause some problems.

It would only be a problem if something were parsing human-targeted error output (which we aren't, and we only guarantee this tool's behavior for our use currently), and relying on the *exact* output, and were using a base revision other than master.